src/components/screens/IndexScreen/Hero.js

```diff
@@ -1,176 +1,360 @@
+  justify-content: center;
+  gap: 8px;
+`;
 
 const Stats = styled.div`
   display: flex;
-  gap: 40px;
+  flex-direction: row;
+  gap: 24px;
+
+  @media (min-width: 940px) {
+    gap: 40px;
+  }
+`;
+
+const HideDesktop = styled.div`
+  display: block;
+
+  @media (min-width: 940px) {
+    display: none;
+  }
+`;
+
+const StatHero = styled.a`
+  display: flex;
+  flex-direction: column;
+  gap: 2px;
+  color: #ffffff;
+  font-size: 16px;
+  cursor: pointer;
+  text-decoration: none;
+
+  span {
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 16px;
+    color: rgba(255, 255, 255, 0.6);
+    transition: color 0.2s ease;
+  }
+
+  &:hover {
+    span {
+      color: #ffffff;
+    }
+  }
+`;
+
+const Right = styled.a`
+  display: none;
+  color: rgba(255, 255, 255, 0.6);
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 16px;
+  text-decoration: none;
+  padding-bottom: 4px;
+  transition: color 0.2s ease;
+
+  &:hover {
+    color: #ffffff;
+  }
+
+  @media (min-width: 940px) {
+    display: flex;
+    align-items: end;
+  }
+`;
+
+const EightContainer = styled.div`
+  position: relative;
+  margin-left: 12px;
+  margin-right: 12px;
+  margin-bottom: -4px;
+`;
+
+const StarStyled = styled(motion.svg)`
+  position: absolute;
+  left: 0;
+  top: 0;
 `;
 
-export function Hero({ contributorCount, npmDownloads, startOpen, ...props }) {
+const Star = ({ x = 0, y = 0, w = 14, delay = 0 }) => {
+  return (
+    <StarStyled
+      initial={{ x, y, opacity: 0, scale: 0.5 }}
+      animate={{ rotate: 360, opacity: 1, scale: 1 }}
+      transition={{ duration: 2, repeat: Infinity, delay }}
+      style={{ originX: `${w / 2}px`, originY: `${w / 2}px` }}
+      width={w}
+      height={w}
+      viewBox="0 0 14 14"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M7 0L8.89064 5.10936L14 7L8.89064 8.89064L7 14L5.10936 8.89064L0 7L5.10936 5.10936L7 0Z"
+        fill="url(#paint0_linear_195_11225)"
+      />
+      <defs>
+        <linearGradient
+          id="paint0_linear_195_11225"
+          x1="7"
+          y1="0"
+          x2="7"
+          y2="14"
+          gradientUnits="userSpaceOnUse"
+        >
+          <stop stopColor="white" />
+          <stop offset="1" stopColor="white" stopOpacity="0" />
+        </linearGradient>
+      </defs>
+    </StarStyled>
+  );
+};
+
+export function Hero({ contributorCount, npmDownloads }) {
+  const [state, setState] = useState(false);
   const { latestVersion, urls = {} } = useSiteMetadata();
-  const { docs = {}, gitHub = {} } = urls;
+  const { docs = {}, gitHub = {}, npm } = urls;
+
+  let npmDownloadsFixed = parseInt((npmDownloads / 1000).toFixed(0), 10);
+  let npmDownloadsDisplay = `${npmDownloadsFixed}k`;
+  if (npmDownloadsFixed >= 1000) {
+    npmDownloadsFixed = (npmDownloadsFixed / 1000).toFixed(2);
+    npmDownloadsDisplay = `${npmDownloadsFixed}m`;
+  }
+
+  const onClick = () => {
+    copy('npx storybook@latest init');
```

Comment:
Instead of a new package (`copy-to-clipboard`), you can use `useCopyToClipboard` from `usehooks-ts`, which we already depend on. Here's an [example usage](https://github.com/storybookjs/frontpage/blob/main/src/components/basics/CodeSnippets/BaseCodeSnippet/SnippetCopyButton/index.tsx#L31).

Reply:
Ah yes good call. I'll replace that

Reply:
Fixed and package removed.